tests/Neo.UnitTests/Neo.UnitTests.csproj

```
@@ -12,6 +12,9 @@
   </ItemGroup>
 
   <ItemGroup>
+    <None Update="SmartContract\Manifest\TestFile\**">
```

```suggestion
    <None Update="SmartContract\Manifest\Artifacts\**">
```

use artifacts folder?

---

Artifacts is in .gitignore, I can't add files to it.